scripts/funcTests/runFuncTests.sh

```diff
@@ -18,11 +18,12 @@ pushd $DIR
 # Download the CLI install script to cli
 echo "Installing dotnet CLI"
 mkdir -p cli
-curl -o cli/dotnet-install.sh https://raw.githubusercontent.com/dotnet/cli/d2bbe1faa294012cec60b640e6522e0674224d3f/scripts/obtain/dotnet-install.sh
+curl -o cli/dotnet-install.sh https://raw.githubusercontent.com/dotnet/cli/4bd9bb92cc3636421cd01baedbd8ef3e41aa1e22/scripts/obtain/dotnet-install.sh
 
 # Run install.sh
 chmod +x cli/dotnet-install.sh
-cli/dotnet-install.sh -i cli -c preview -v 1.0.1
+# cli/dotnet-install.sh -i cli -c 2.0 --version 2.0.2
```

Comment:
This caused problems, but when we move to netstandard2.0 we will need so this so I'm leaving it commented out for now